src/epd2in13_v2/mod.rs

```diff
@@ -0,0 +1,558 @@
+
+/// EPD2in13 (V2) driver
+///
+pub struct EPD2in13<SPI, CS, BUSY, DC, RST> {
+    /// Connection Interface
+    interface: DisplayInterface<SPI, CS, BUSY, DC, RST>,
+
+    sleep_mode: DeepSleepMode,
+
+    /// Background Color
+    background_color: Color,
+    refresh: RefreshLUT,
+}
+
+impl<SPI, CS, BUSY, DC, RST> InternalWiAdditions<SPI, CS, BUSY, DC, RST>
+    for EPD2in13<SPI, CS, BUSY, DC, RST>
+where
+    SPI: Write<u8>,
+    CS: OutputPin,
+    BUSY: InputPin,
+    DC: OutputPin,
+    RST: OutputPin,
+{
+    fn init<DELAY: DelayMs<u8>>(
+        &mut self,
+        spi: &mut SPI,
+        delay: &mut DELAY,
+    ) -> Result<(), SPI::Error> {
+        // HW reset
+        self.interface.reset(delay);
+
+        if self.refresh == RefreshLUT::QUICK {
+            self.set_vcom_register(spi, (-9).vcom())?;
+            self.wait_until_idle();
+
+            self.set_lut(spi, Some(self.refresh))?;
+
+            // Python code does this, not sure why
+            // self.cmd_with_data(spi, Command::WRITE_OTP_SELECTION, &[0, 0, 0, 0, 0x40, 0, 0])?;
+
+            // During partial update, clock/analog are not disabled between 2
+            // updates.
+            self.set_display_update_control_2(
+                spi,
+                DisplayUpdateControl2::new().enable_analog().enable_clock(),
+            )?;
+            self.command(spi, Command::MASTER_ACTIVATION)?;
+            self.wait_until_idle();
+
+            self.set_border_waveform(
+                spi,
+                BorderWaveForm {
+                    vbd: BorderWaveFormVBD::GS,
+                    fix_level: BorderWaveFormFixLevel::VSS,
+                    gs_trans: BorderWaveFormGS::LUT1,
+                },
+            )?;
+        } else {
+            self.wait_until_idle();
+            self.command(spi, Command::SW_RESET)?;
+            self.wait_until_idle();
+
+            self.set_driver_output(
+                spi,
+                DriverOutput {
+                    scan_is_linear: true,
+                    scan_g0_is_first: true,
+                    scan_dir_incr: true,
+                    width: (HEIGHT - 1) as u16,
+                },
+            )?;
+
+            self.set_data_entry_mode(
+                spi,
+                DataEntryModeIncr::X_INCR_Y_INCR,
+                DataEntryModeDir::X_DIR,
+            )?;
+
+            // Use simple X/Y auto increase
+            self.set_ram_area(spi, 0, 0, WIDTH - 1, HEIGHT - 1)?;
+            self.set_ram_address_counters(spi, 0, 0)?;
+
+            self.set_border_waveform(
+                spi,
+                BorderWaveForm {
+                    vbd: BorderWaveFormVBD::GS,
+                    fix_level: BorderWaveFormFixLevel::VSS,
+                    gs_trans: BorderWaveFormGS::LUT3,
+                },
+            )?;
+
+            self.set_vcom_register(spi, (-21).vcom())?;
+
+            self.set_gate_driving_voltage(spi, 190.gate_driving_decivolt())?;
```

Comment:
I really don't know. I've already skipped most of the calls that simply sets the value to the reset values... You mean having a const like `VCOM_DEFAULT = -21` and use it here instead of the `(-21)` ?

Reply:
Yes, but after thinking about it once more it doesn't make much sense.

Reply:
Ok! np :)